pkg/cmd/server/admin/create_kubeconfig.go

```diff
@@ -161,15 +160,15 @@ func (o CreateKubeConfigOptions) CreateKubeConfig() (*clientcmdapi.Config, error
 	}
 
 	// determine all the nicknames
-	clusterNick, err := cliconfig.GetClusterNicknameFromURL(o.APIServerURL)
+	clusterNick, err := config.GetClusterNicknameFromURL(o.APIServerURL)
```

Comment:
I don't think I care what this nickname is.  Open an issue to discuss using ugly names that assume an immutable file.

Reply:
> I don't think I care what this nickname is. Open an issue to discuss using ugly names that assume an immutable file.

Tracking this here: https://github.com/openshift/origin/issues/17485